src/Modules/Guilds/pages/CreateProposal.tsx

```diff
@@ -77,6 +86,11 @@ const CreateProposalPage: React.FC = () => {
   const [isIpfsErrorModalOpen, setIsIpfsErrorModalOpen] = useState(false);
   const [skipUploadToIPFs, setSkipUploadToIPFs] = useState(false);
 
+  const isActionDenied = useMemo(() => {
+    return !!options.find(option =>
+      option.decodedActions.find(action => action.actionDenied === true)
+    );
+  }, [options]);

```

Comment:
Since your interest to find at least 1 actionDenied I would say `some` is better here to stop looping. 
```suggestion
      const isActionDenied = useMemo(() => options.some(({ decodedActions }) =>
          decodedActions.some(({ actionDenied }) => !!actionDenied)),
        [options]
      );
```